scss/civicrm/event/_detail.scss

```diff
@@ -0,0 +1,243 @@
+#crm-container {
+  input[type=radio]:not(.old-radiocheckbox-style):checked+label[for]::after {
+    top: 7px;
+  }
+
+  button {
+    background: $brand-primary;
+    border: 0;
+    border-radius: $border-radius-base;
+    color: $crm-pure-white-color;
+    font-family: $font-family-base;
+    height: 36px;
+    line-height: 1;
+    padding: 11px 12px;
+    text-shadow: none;
+    text-transform: uppercase;
+    vertical-align: middle;
+
+    &:hover {
+      background: darken($brand-primary, $crm-darken-percentage);
+    }
+
+    &:active,
+    &:focus {
+      outline: none;
+    }
+
+    &.cancel {
+      background: $crm-white;
+      border: 1px solid $crm-copy;
+      color: $crm-cancel-color;
+
+      &:hover {
+        background: $crm-copy;
+        color: $crm-white;
+      }
+    }
+  }
+
+  .crm-actions-ribbon {
+    margin-bottom: 20px;
+  }
+
+  .crm-form-block.crm-form-block>.crm-submit-buttons,
+  .crm-form-block .form-item>.crm-submit-buttons {
+    border-top: 0;
+    margin-top: 0 !important;
+  }
+
+  #newLocation {
+    padding: 0 20px 20px 15px;
+
+    #crm-submit-buttons {
```

Comment:
same as above

Reply:
Just added additional gutter
![additional-gutter](https://user-images.githubusercontent.com/3272950/34294920-652a591a-e713-11e7-933a-97aa6573a9f3.png)

Reply:
@boriszaydlin why is the gutter needed in the first place? is this an exception that we are adding for this particular page (and if so, why), or is this some common pattern?

Reply:
We need 20px left gutter. I added only 15px to #newLocation wrapper, because forms wrapped in table where all <td> already have 5px left gutter. So I added additional 5px for buttons too